core/src/main/scala/kafka/tools/ConsoleConsumer.scala

```diff
@@ -103,6 +103,8 @@ object ConsoleConsumer extends Logging {
         consumer.stop()
 
         shutdownLatch.await()
+
+        System.out.println("shutdown_complete")
```

Comment:
Is there a risk of breaking tools which depend on console consumer to read messages? You'd need to filter this message if you're doing any processing on the results. Would it be better to add an option to console consumer to explicitly enable this? For example, "--enable-lifecycle-logging."